src/SFA.DAS.Reservations.Functions.LegalEntities.AcceptanceTests/Features/LegalEntityUpdated.feature

```diff
@@ -0,0 +1,23 @@
+﻿Feature: Legal Entity Updated
+	In order know when a legal entity has been updated
+	I want to be shown when its latest state
+
+Scenario: A existing legal entity has been signed
+	Given I have an existing unsigned, non levy legal entity
+	When signed agreement event is triggered
+	Then the legal entity should be signed
+
+Scenario: A existing legal entity has recieved levy
+	Given I have an existing unsigned, non levy legal entity
+	When levy added event is triggered
+	Then the legal entity should be marked as a levy entity
+
+Scenario: A new legal entity has recieved levy
+	Given I have a legal entity that is new
+	When levy added event is triggered
+	Then the legal entity should not be available
```

Comment:
Not sure these last two Thens are really describing the business functionality

Reply:
I couldn't come up with a good description that matched both what was happening and what the business output was but i'll have a rethink and see if I can come up with something better